app/helpers/chargeback_rate_detail_currency_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ChargebackRateDetailCurrencyHelper
+  def currencies_for_select
+    # Return an array with the codes of the currencies
+    currency_id = []
+    currency_code = []
+    ChargebackRateDetailCurrency.all.each do |i|
+      currency_code << i.symbol + " [" + i.full_name + "]"
```

Comment:
Better still - WDYT about making this whole thing a class method on `ChargebackRateDetailCurrency` as well?